zipline/utils/test_utils.py

```diff
@@ -356,3 +361,57 @@ class ExplodingObject(object):
     """
     def __getattribute__(self, name):
         raise UnexpectedAttributeAccess(name)
+
+
+class tmp_assets_db(object):
+    """Create a temporary assets sqlite database.
+    This is meant to be used as a context manager.
+
+    Paramaters
+    ----------
+    data : dict, optional
+        The data to feed to the writer. By default this maps:
+        ('A', 'B', 'C') -> map(ord, 'ABC')
+    """
+    def __init__(self, data=None):
+        self._eng = None
+        self._data = AssetDBWriterFromDictionary(
```

Comment:
Sure, I will see if they are easily translatable. One thing I like is that this format is really easy to pass in at a call site though.